src/blog/2024/07/deploying-flowfuse-with-docker.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Deploying FlowFuse with Docker on an Ubuntu server
+subtitle: Step-by-step guide on how to deploy FlowFuse with Docker on Ubuntu server
+description: Learn to deploy the FlowFuse on your Ubuntu server with Docker, from domain setup to running your app with email, SSL configurations.
+date: 2024-07-15
```

Comment:
@sumitshinde-84 was it your plan to publish it next week?

Reply:
feel free to update it !

Reply:
Yes, I usually update dates once they receive at least one approval